src/shared/telemetry/activation.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { AwsContext } from '../awsContext'
+import { SettingsConfiguration } from '../settingsConfiguration'
+import { DefaultTelemetryService } from './defaultTelemetryService'
+import { ext } from '../extensionGlobals'
+import { getLogger } from '../logger'
+import { TelemetryService } from './telemetryService'
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+// SETTINGS_TELEMETRY_VALUE_XXX must be kept in sync with package.json "aws.telemetry" configuration values
+const SETTINGS_TELEMETRY_VALUE_ENABLE = 'Enable'
+const SETTINGS_TELEMETRY_VALUE_DISABLE = 'Disable'
+const SETTINGS_TELEMETRY_VALUE_USEIDE = 'Use IDE settings'
+
+export const responseEnable = localize('AWS.telemetry.notificationYes', 'Enable')
+export const responseDisable = localize('AWS.telemetry.notificationNo', 'Disable')
```

Comment:
Related to the above: if those aren't translated, we may want to keep these in English in order to keep our messaging straight (since the Settings UI will only show English)

Reply:
These buttons were left unchanged in this PR. It makes more sense to have these translated, as they're prompting a user to take action.

An upcoming change will alter this notification, so this is moot.